### PR TITLE
Some improvements…

### PR DESCRIPTION
**Pull request by SkYNewZ**
- Give a Github issue template
- Set up GitHub Action to check code lint using [Black](https://pypi.org/project/black/)
- Upgrade `LICENSE` to year 2020
- Move binaries to their folder
- Convert to a Python module using `setuptools` and move credentials file and input file to root
- Regarding some issues, I decided to commit `input.txt` to give an example to users
- Because of the new Facebook design coming, [fix](https://github.com/harismuneer/Ultimate-Facebook-Scraper/pull/74/files#diff-a006272bcea5455c6e82c20173e365d0L817-R832) the `login` button

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or command output**
+If applicable, add screenshots or `output logs` to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Linux]
+ - Python version [e.g. 3.7.5]
+ - Version [e.g. 0a5d89e] (run `git rev-parse --short HEAD` to get it)
```
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

**Comment by harismuneer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           

Complexity increasing per file
==============================
- scraper/scraper.py  37
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Ultimate-Facebook-Scraper/Ultimate-Facebook-Scraper/pullRequest?prid=4591623&bid=15416535)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or command output**
+If applicable, add screenshots or `output logs` to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Linux]
+ - Python version [e.g. 3.7.5]
```
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or command output**
+If applicable, add screenshots or `output logs` to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Linux]
```
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (harismuneer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

Comment (SkYNewZ) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a Github template

**Comment by harismuneer** [comment]
Awesome job @SkYNewZ!

**Pull request merged**